website/source/docs/providers/aws/r/lightsail_domain.html.markdown

```diff
@@ -14,7 +14,7 @@ a domain name using Amazon Route 53 or another domain name registrar.
 If you have already registered your domain, you can enter its name in
 this parameter to manage the DNS records for that domain.
 
-~> **Note:** Lightsail is currently only supported in `us-east-1` region.
+~> **Note:** Lightsail is currently only supported in `us-east-1`, `us-east-2`, `us-west-2`, `eu-west-1`, `eu-west-2`, `eu-central-1` regions.
```

Comment:
Rather than playing catch-up as Amazon continue to add regions to Lightsail - I think it'd make more sense to update this to link to a list of supported regions at Amazon's end? I've found [this document](https://lightsail.aws.amazon.com/ls/docs/overview/article/understanding-regions-and-availability-zones-in-amazon-lightsail) which defines the supported regions - as such could we change this to something like:

```
~> **Note:** Lightsail is currently only supported in a limited number of AWS Regions, please see ["Understanding Regions and Availability Zones in Amazon Lightsail"](https://lightsail.aws.amazon.com/ls/docs/overview/article/understanding-regions-and-availability-zones-in-amazon-lightsail) for more details
```